### DISCUSSION TITLE
ArgumentError: user %n doesn't exist

### DISCUSSION THREAD

**Discussion by MBurchard** [Homebrew/discussions#3882] [Everyday usage]
Hi, during installation of`kubelogin` I got the error `ArgumentError: user %n doesn't exist`

I opened the following issue, but they said it's a homebrew thing: https://github.com/Azure/kubelogin/issues/148

As I have already written there, I don't know which process has created the evnironment variable `USER` and set it to `%n`.

I'm new to homebrew and can't say, witch value this variable should have. Of course, I can just reset it.
It would be really nice if someone could help me with this issue.

**Comment by Bo98** [comment]
This is from Ruby itself when `USER` isn't a valid value. What shell are you using? It should automatically be set to the account name.

Try check `~/.bash_profile`, `~/.zprofile`, `~/.zshrc` etc for anything that might be overwriting it.

**Comment by MBurchard** [comment]
I'm using zsh combined with oh-my-zsh...
I checked the both zsh files and no file contains `USER`. A `.bash_profile` does not exists.
